### PR TITLE
i love commas man

### PR DESCRIPTION
**Pull request by efefury**
added commas to hotm pv tab and to recipe viewer

### CODE REVIEW

File: src/main/java/io/github/moulberry/notenoughupdates/profileviewer/GuiProfileViewer.java [hunk 1]
```diff
@@ -3933,7 +3933,7 @@ private void drawMiningPage(int mouseX, int mouseY, float partialTicks) {
 				EnumChatFormatting.GRAY + "Ore.",
 				"",
 				EnumChatFormatting.GRAY + "Cost",
-				EnumChatFormatting.DARK_GREEN + "" + (int) Math.pow(tittyInsane + 2, 3.1) + " Mithril Powder"
+				EnumChatFormatting.DARK_GREEN + "" + NumberFormat.getInstance().format((int) Math.pow(miningSpeed + 2, 3)) + " Mithril Powder"
```
Comment (romangraef) [inline src/main/java/io/github/moulberry/notenoughupdates/profileviewer/GuiProfileViewer.java:3936]: why you change variable from titty to minignspeed

**Review by romangraef — CHANGES_REQUESTED** [review]
isnt java util numberformatter locale dependent (which could result in formatting inconsistent with hypixels(this is especially a concern with the coins)) so id prefer if we just didnt do this pr at all like this (although i am very tired so maybe im confusion stuff)

File: src/main/java/io/github/moulberry/notenoughupdates/profileviewer/GuiProfileViewer.java (hunk 1 above)
Comment (jani270) [reply src/main/java/io/github/moulberry/notenoughupdates/profileviewer/GuiProfileViewer.java:3936]: ctrl c ctrl v moment

Comment (efefury) [reply src/main/java/io/github/moulberry/notenoughupdates/profileviewer/GuiProfileViewer.java:3936]: i didnt see that one i am blind i just copy pasted i didnt really look at it before pr'ing

**Review by NopoTheGamer — APPROVED** [review]

**Review by Lulonaut — APPROVED** [review]

**Review by romangraef — APPROVED** [review]

**Pull request merged**